galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -327,7 +327,7 @@ public void runTest(Properties bootstrapProperties, Properties overridePropertie
         try {
             if (managers.anyReasonTestClassShouldBeIgnored()) {
                 stopHeartbeat();
-                updateStatus("finished", "finished");
+                updateStatus(TestRunLifecycleStatus.FINISHED, "finished");
                 frameworkInitialisation.shutdownFramework();
                 return; // TODO handle ignored classes
```

Comment:
Do we need all these return statements in the middle of the method ?

Reply:
No, however due to having no unit tests for this logic, we shall deal with this in a specific refactoring issue.